File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter02/JAVASCRIPT/js/compareSum02.js
function calculate(){
    let num1=0;   //numero1
    let num2=0;   //numero2
    let num3=0;   //numero3
    let sum1=0;   //suma de numero1+numero2
    let sum2=0;   //suma de numero1+numero3
    let sum3=0;   //suma de numero2+numero3

    let paragrafg=document.createElement('p');
    paragrafg.setAttribute("id", "idParagrafg");
        if(document.getElementById("idBtnCalculate").value=="Calcular"){
            
            num1=parseInt(document.getElementById("idNumber01").value);
            num2=parseInt(document.getElementById("idNumber02").value);
            num3=parseInt(document.getElementById("idNumber03").value);
            
            sum1=num1+num2;
            sum2=num1+num3;
            sum3=num2+num3;
            
            paragrafg.innerHTML+="Suma 1: "+num1+" + "+num2 + " = "+sum1+"<p></p>";
            paragrafg.innerHTML+="Suma 2: "+num1+" + "+num3 + " = "+sum2+"<p></p>";
            paragrafg.innerHTML+="Suma 3: "+num2+" + "+num3 + " = "+sum3+"<p></p>";
            if(sum1==num3){
                paragrafg.innerHTML+="Suma 1: "+sum1+" es igual al numero: "+3;
            }else if(sum2==num2){
                paragrafg.innerHTML+="Suma 2: "+sum2+" es igual al numero: "+2;
            }else if(sum3==num1){
                paragrafg.innerHTML+="Suma 3: "+sum3+" es igual al numero: "+1;
            }else{
                paragrafg.innerHTML+="Son Distintos";
            }

            document.getElementById("idSection").appendChild(paragrafg);
            document.getElementById("idBtnCalculate").value="Limpiar";
        }else{
            document.getElementById("idNumber01").value="";
            document.getElementById("idNumber02").value="";
            document.getElementById("idNumber03").value="";
            
            document.getElementById("idParagrafg").remove();
        
            document.getElementById("idBtnCalculate").value="Calcular";
        }
    

}

File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter02/CSHARP/CompareSum02.cs
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 02.-Metodología de la programación y desarrollo de software.
    Problema 24. Programa que lea tres numeros y descubra si uno de ellos es la suma de los otros dos
*/
using System;
class CompareSum{
    public static void Main(string[] args){
        int num1=0;   //numero1
        int num2=0;   //numero2
        int num3=0;   //numero3
        int sum1=0;   //suma de numero1+numero2
        int sum2=0;   //suma de numero1+numero3
        int sum3=0;   //suma de numero2+numero3
        System.Console.Write("\nIngresa el primer numero: ");
        num1=int.Parse(System.Console.ReadLine());
        System.Console.Write("Ingresa el segundo numero: ");
        num2=int.Parse(System.Console.ReadLine());
        System.Console.Write("Ingresa el tercer numero: ");
        num3=int.Parse(System.Console.ReadLine());
        sum1=num1+num2;
        sum2=num1+num3;
        sum3=num2+num3;
        if(sum1==num3){
            System.Console.Write("\nSon iguales\n");
        }else if(sum2==num2){
            System.Console.Write("\nSon iguales\n");
        }else if(sum3==num1){
            System.Console.Write("\nSon iguales\n");
        }else{
            System.Console.Write("\nSon Distintos\n");
        }    
    }
}